gatsby-node.js

```diff
@@ -42,6 +42,107 @@ const getMessages = (path, language) => {
   }
 }
 
+/**
+ * Markdown isOutdated check
+ * Parse headers in markdown file (both translated and english) and compare their info structure.
+ * If this structure is not the same, then the file isOutdated.
+ * If there is not english file, return true
+ * @param {string} path filepath for translated mdx file
+ * @returns boolean for if file is outdated or not
+ */
+const checkIsMdxOutdated = (path) => {
+  try {
+    const splitPath = path.split("/")
+    splitPath.splice(7, 2)
+    const englishPath = splitPath.join("/")
+
+    const re = /([#]+) [^\{]+\{#([^\}]+)\}/gim
+
+    const translatedData = fs.readFileSync(path, "utf-8")
+    const englishData = fs.readFileSync(englishPath, "utf-8")
+
+    let englishMatch = ""
+    let intlMatch = ""
+    englishData.match(re).forEach((match) => {
+      englishMatch += match.replace(re, (_, p1, p2) => p1 + p2)
+    })
+    translatedData.match(re).forEach((match) => {
+      intlMatch += match.replace(re, (_, p1, p2) => p1 + p2)
+    })
+
+    return englishMatch !== intlMatch
+  } catch (err) {
+    return true
+  }
+}
+
+/**
+ * JSON isOutdated check
+ * Checks if translated JSON file exists.
+ * If translated file exists, checks that all translations are present (checks keys), and that all the keys are the same.
+ * If translation file exists, isContentEnglish will be false
```

Comment:
Is there a difference between `translated file` & `translation file`? If not let's use the same term.